wallet/src/components/Proposal.tsx

```diff
@@ -272,7 +278,13 @@ const Proposal = ({ offer, purses, swingsetParams, beansOwing }) => {
       {error && (
         <div className="OfferEntry">
           <h6>Error</h6>
-          <details style={{ whiteSpace: 'pre-wrap' }}>{error}</details>
+          <details
+            style={{
+              whiteSpace: 'pre-wrap',
+            }}
+          >
+            <Box sx={{ fontFamily: '"RobotoMono", monospace' }}>{error}</Box>
```

Comment:
the error message may be more readable as proportional font. maybe with a touch of red.

Reply:
Okay just changed it back, didn't go as far as red though because I'm not sure about it